content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title:
+  'Making observability fun: How we increased engineers confidence in incident
+  management using a game'
+linkTitle: Observability game day
```

Comment:
I presume this is the title that appears on the side bar. Should it mention Skyscanner or the OTel Demo?

Reply:
I have updated the linkTitle :)